### PR TITLE
[Pages]: Update custom branches section.

### PR DESCRIPTION
**Pull request by Ekwuno**
This PR updates the custom branches section of our git integration to reflect the branch to build control option.

### CODE REVIEW

File: content/pages/platform/git-integration.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ Cloudflare supports connecting Cloudflare Pages to your GitHub and GitLab reposi
...
+Suppose you have a custom Git workflow that uses specific branches to represent your project's production build. In that case, you can specify a custom branch when creating (or managing an existing) project in the Pages dashboard by going to  **Settings** > **Builds & deployments** and clicking the **Configure Production deployments** button.
 
-For example, imagine that you merge new code into the `main` branch. When it is time to deploy your project, you merge a set of commits into the `production` branch. In the Cloudflare Pages dashboard, you can set the default branch to `production`.
+For example, imagine that you merge new code into your GitHub `main` branch. When it is time to deploy your project to Pages, your merged set of commits will be deployed to the `production` branch in the Cloudflare Pages dashboard. You can set the default build branch to `production` or change it to any other branch by going to the Pages project's **Settings** > **Builds & deployments** > **Configure Production deployment**. In the dropdown menu under **production branch**, select the new `production` branch.
```
Comment (WalshyDev) [inline content/pages/platform/git-integration.md]: This sounds like we will merge the code from `main` -> `production`

Also we're stating how to change the production branch twice, we already told them how to change it. We should either say it above or say it here, it's weird to repeat it.

File: content/pages/platform/git-integration.md [hunk 2]
```diff
@@ -9,11 +9,14 @@ Cloudflare supports connecting Cloudflare Pages to your GitHub and GitLab reposi
...
+
+You can also use [preview deployments](/pages/platform/preview-deployments/) to preview how the new version of your project looks before merging to `production`. In addition, Pages allows you to configure how your preview branches are built by using [branch build controls](/pages/platform/branch-build-controls/), which will enable you to configure automatic deployments to suit your preference on a per-project basis.
+
+To do this in your Pages project's go-to  **Settings** > **Builds & deployments** > **Configure preview deployment** and select **custom branches** you can specify branches you wish to include and exclude from automatic deployments in the provided configuration fields. To learn more refer to the [branch build controls](/pages/platform/branch-build-controls/) documentation.
```
Comment (WalshyDev) [inline content/pages/platform/git-integration.md]:
```suggestion
To configure this in your Pages project go to **Settings** > **Builds & deployments** > **Configure preview deployment** and select **Custom branches**. Here you can specify branches you wish to include and exclude from automatic deployments in the provided configuration fields. To learn more refer to the [branch build controls](/pages/platform/branch-build-controls/) documentation.
```

File: content/pages/platform/git-integration.md [hunk 3]
```diff
@@ -9,11 +9,14 @@ Cloudflare supports connecting Cloudflare Pages to your GitHub and GitLab reposi
...
 
-For example, imagine that you merge new code into the `main` branch. When it is time to deploy your project, you merge a set of commits into the `production` branch. In the Cloudflare Pages dashboard, you can set the default branch to `production`.
+For example, imagine that you merge new code into your GitHub `main` branch. When it is time to deploy your project to Pages, your merged set of commits will be deployed to the `production` branch in the Cloudflare Pages dashboard. You can set the default build branch to `production` or change it to any other branch by going to the Pages project's **Settings** > **Builds & deployments** > **Configure Production deployment**. In the dropdown menu under **production branch**, select the new `production` branch.
+
+You can also use [preview deployments](/pages/platform/preview-deployments/) to preview how the new version of your project looks before merging to `production`. In addition, Pages allows you to configure how your preview branches are built by using [branch build controls](/pages/platform/branch-build-controls/), which will enable you to configure automatic deployments to suit your preference on a per-project basis.
```
Comment (WalshyDev) [inline content/pages/platform/git-integration.md]:
```suggestion
You can also use [preview deployments](/pages/platform/preview-deployments/) to preview how the new version of your project looks before merging into `production`. In addition, Pages allows you to configure which of your preview branches are built and deployed by using [branch build controls](/pages/platform/branch-build-controls/).
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: content/pages/platform/git-integration.md (hunk 1 above)
Comment (Ekwuno) [reply content/pages/platform/git-integration.md]: Yea the naming isn't the best. However, I think clarifying " GitHub `main` branch and  " the `production` branch in the Cloudflare Pages"  helps with the confusion. 


I will try to rephrase this section

**Review by jkup — APPROVED** [review]
Looks good to me!  A few thoughts:

1. Would it make sense to put this lower down on the page? I was expecting to see the "​​Organizational access" stuff before Custom branches. I realize it's always been that was I just thought it was surprising.
2. Might be cool to add a screenshot of the "Custom Branches" page! Totally up to you!

**Pull request merged**